uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/ui/views/HistoryChart.kt

```diff
@@ -190,7 +199,9 @@ class HistoryChart(
     ) {
 
         val value = if (offset >= series.size) Square.OFF else series[offset]
+        val notes = if (offset >= hasNotes.size) false else hasNotes[offset]
```

Comment:
In at least part of the code, `notes` refer to a string variable, it'd be nice to have more consistency in the variable names.

Reply:
I could change the variable name `notes` which refers boolean array to `notesIndicators` or maybe you could suggest something.